LICENSE.md

```diff
@@ -0,0 +1,37 @@
+## Part of this project is not in the public domain
+
+The Roboto Mono font files in `Sketch/fonts` are from [Roboto](https://github.com/google/roboto) by Google, licensed under [Apache v2.0](https://github.com/google/roboto/blob/master/LICENSE).
```

Comment:
I removed the 18Franklin notice, as the 18Franklin font is no longer in this repo.